tests/eas/acceptance.py

```diff
@@ -395,17 +304,27 @@ def test_big_cpus_fully_loaded(self):
 
             self.assertGreater(busy_time, expected_busy_time, msg=msg)
 
-    def test_little_cpus_run_tasks(self):
+    @experiment_test
+    def test_little_cpus_run_tasks(self, experiment, tasks):
         """Offload Migration and Idle Pull: Little cpus run tasks while bigs are busy"""
-        tasks = self.params.keys()
+
         num_offloaded_tasks = len(tasks) / 2
 
-        first_task_finish_time = min(self.end_times.values())
+        end_times = self.calculate_end_times(experiment).values()
+        first_task_finish_time = min(end_times)
+
+        migrators_assert = self.get_multi_assert(experiment, "migrator")
+        start_time = min(t["starttime"]
```

Comment:
For start times we can probably have the same concept of EndTime.
Although I'm not completely sure we can populate it in a generic way...

If it should be possible than perhaps we should defined a more generic
```python
Times = namedtuple("Times", ['values', 'first', 'last'])
```
to be used both for start and end times ;-)